main.go

```diff
@@ -155,6 +155,11 @@ func main() {
 		log.Fatal("Unauthorized: No GitHub token present")
 	}
 
+	cdqToken := os.Getenv("CDQ_GITHUB_TOKEN")
```

Comment:
I would add a quick little note in the README

Reply:
updated